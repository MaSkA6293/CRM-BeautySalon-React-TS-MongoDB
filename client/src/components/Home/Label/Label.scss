.Label {
  // background-color: transparent;
  // flex: 0 0 10%;
  // align-content: center;
  // display: flex;
  // align-items: center;
  // flex-flow: column;
  // justify-content: center;
  h1 {
    text-align: center;
    color: #7f006c;
  }
}
