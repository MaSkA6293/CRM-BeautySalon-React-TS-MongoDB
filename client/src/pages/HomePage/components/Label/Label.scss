.Label {
  margin: 10px 0;
  h1 {
    text-align: center;
    color: #7f006c;
  }
}
