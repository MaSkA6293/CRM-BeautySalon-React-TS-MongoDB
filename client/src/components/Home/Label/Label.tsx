import React from "react";
import "./Label.scss";
import classNames from "classnames";

type LabelProps = {
  title: string;
  addClass?: string[];
};

const Label = ({ title, addClass }: LabelProps) => {
  return (
    <div className={classNames("Label", addClass)}>
      <h1>{title}</h1>
    </div>
  );
};

export default Label;
