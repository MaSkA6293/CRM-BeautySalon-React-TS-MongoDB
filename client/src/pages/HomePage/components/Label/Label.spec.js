import React from "react";
import Label from "./index";

const setUp = (props) => shallow(<Label {...props} />);

describe("all tests Label", () => {
  let component;
  beforeEach(() => {
    component = setUp();
  });
  describe("Label component", () => {
    it("should render Label component", () => {
      expect(component).toMatchSnapshot();
    });
  });
});
