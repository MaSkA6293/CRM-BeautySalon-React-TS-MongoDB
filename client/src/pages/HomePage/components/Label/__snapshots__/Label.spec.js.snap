// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`all tests Label Label component should render Label component 1`] = `
<div
  className="Label"
>
  <h1 />
</div>
`;
